server/mlabns/handlers/lookup.py

```diff
@@ -12,54 +13,17 @@
 from google.appengine.ext.webapp import template
 
 
-def _create_af_aware_fqdn(fqdn, address_family):
-    """Adds the v4/v6 only annotation to the fqdn.
-
-    Example:
-        fqdn:       'npad.iupui.mlab3.ath01.measurement-lab.org'
-        ipv4 only:  'npad.iupui.mlab3v4.ath01.measurement-lab.org'
-        ipv6 only:  'npad.iupui.mlab3v6.ath01.measurement-lab.org'
-
-    Args:
-        fqdn: A tool FQDN with no address family specific annotation.
-        address_family: The address family for which to create the FQDN or None
-            to create an address family agnostic FQDN.
-
-    Returns:
-        A FQDN specific to a particular address family, or the original FQDN
-        if an address family is not specified.
-    """
-    if address_family == message.ADDRESS_FAMILY_IPv4:
-        fqdn_annotation = 'v4'
-    elif address_family == message.ADDRESS_FAMILY_IPv6:
-        fqdn_annotation = 'v6'
-    elif not address_family:
-        fqdn_annotation = ''
-    else:
-        logging.error('Unrecognized address family: %s', address_family)
-        return fqdn
-
-    fqdn_parts = fqdn.split('.')
-    fqdn_parts[2] += fqdn_annotation
-
-    return '.'.join(fqdn_parts)
-
-
-def _create_tool_url(fqdn, address_family, http_port):
+def _create_tool_url(fqdn, http_port):
     """Create an HTTP URL for a tool.
 
     Args:
-        fqdn: A tool FQDN with no address family specific annotation.
-        address_family: The address family for which to create the URL or None
-            to create an address family agnostic URL.
+        fqdn: A tool FQDN (already processed by fqdn_rewrite).
         http_port: The HTTP port number part of the URL.
 
     Returns:
-        A tool URL, annotated with the appropriate address family (when
-        available).
+        A constructed tool HTTP URL with protocol and http port.
     """
-    return 'http://%s:%s' % (_create_af_aware_fqdn(fqdn, address_family),
-                             http_port)
+    return 'http://%s:%s' % (fqdn, http_port)
```

Comment:
I don't think we are disagreeing here.  My original point, spread across two comments, was that returning http:7123 to an "ndt_ssl" request didn't seem right.  Removing the port 7123 from the URL is a start, but still returning an http "url" (even minus the port) is also misleading, because the it's not even a valid URL.  That is, the browser will try to connect on port 80, which will fail.  I think we shouldn't publish any "url" at all for "ndt_ssl" requests until such a time as we have an actual SSL url that works.

Are we saying the same thing here, and have just got lost in the chain of comments?